configure/ols-configuring-openshift-lightspeed.adoc

```diff
@@ -24,7 +24,7 @@ include::modules/ols-about-lightspeed-and-role-based-access-control.adoc[levelof
 include::modules/ols-granting-access-to-individual-users.adoc[leveloffset=+2]
 include::modules/ols-granting-access-to-user-group.adoc[leveloffset=+2]
 include::modules/ols-filtering-and-redacting-information.adoc[leveloffset=+1]
-include::modules/ols-cluster-interaction-overview.adoc[leveloffset=+1]
+include::modules/ols-about-cluster-interaction.adoc[leveloffset=+1]
```

Comment:
Just a quick FYI that you don't have to change the file name if you're just changing the title. It's not as big of a deal with modules since if anyone has it bookmarked, it'll still go to the assembly, just won't jump down to the right section. It's a bigger deal w/ assemblies since it could 404 though. Anyways, not gonna hold up the PR over it, just wanted to mention!